s29741-pj-JakubKolad/2023.09.30/Z1.txt

```diff
@@ -40,7 +40,7 @@ ls
 
 7.Wyświetl zawartość swojego katalogu domowego bez przechodzenia do niego 
 (ls z odpowiednim parametrem).
-ls /home
+ls ~
```

Comment:
No to teraz by wypadało jeszcze napisać jaka jest różnica pomiędzy `/home` a `~`. Na co rozwija się `~` i czy to jest dokładnie to samo co `/home` ?